# MPMI Congress Training 
Here are my slides and references for IS-MPMI Congress 2016 Portland, Oregon Bioinformatics Training.

## Databases

Slides - [databases.pptx](databases.pptx)

Links to databases from the database section of my talk:

1. Ensembl [http://ensembl.org](http://ensembl.org)
2. Ensembl Plants [http://plants.ensembl.org](http://plants.ensembl.org)
3. Ensembl Fungi [http://fungi.ensembl.org](http://fungi.ensembl.org)
4. Ensembl Protists [http://protists.ensembl.org](http://protists.ensembl.org)
5. Ensembl Bacteria [http://bacteria.ensembl.org](http://bacteria.ensembl.org)
6. PhytoPath [http://phytopathdb.org](http://phytopathdb.org)
7. SolGenomics [http://solgenomics.net](http://solgenomics.net)
8. TAIR [http://arabidopsis.org](http://arabidopsis.org)
9. AraPort [http://araport.org](http://araport.org)

## Genome Assembly

Slides - [genome_assembly.ppt](genome_assembly.pptx)

References and Links:

1. Mike Schatz's tutorial slides - [http://schatzlab.cshl.edu/teaching/2012/CSHL.Sequencing/Whole%20Genome%20Assembly%20and%20Alignment.pdf](http://schatzlab.cshl.edu/teaching/2012/CSHL.Sequencing/Whole%20Genome%20Assembly%20and%20Alignment.pdf)
2. Application of 'next-generation' sequencing technologies to microbial genetics - Daniel MacLean, Jonathan D. G. Jones1 & David J. Studholme (2009) Nature Reviews Microbiology 7, 287-296
2. Current challenges in de novo plant genome sequencing and assembly - Schatz MC, Witkowski, McCombie, WR (2012) Genome Biology. 12:243
2. Assembly of Large Genomes using Second Generation Sequencing - Schatz MC, Delcher AL, Salzberg SL (2010) Genome Research. 20:1165-1173.
2. Assembly Complexity of Prokaryotic Genomes using Short Reads. - Kingsford C, Schatz MC, Pop M (2010) BMC Bioinformatics. 11:21.
2. SOAPdenovo [http://soap.genomics.org.cn/soapdenovo.html](http://soap.genomics.org.cn/soapdenovo.html)
2. Quake: quality-aware detection and correction of sequencing reads. - Kelley, DR, Schatz, MC, Salzberg SL (2010) Genome Biology. 11:R116
2. Celera Assembler [http: //wgs-assembler.sf.net](http://wgs-assembler.sf.net]
2. Hybrid error correction and de novo assembly of single-molecule sequencing reads.  - Koren, S, Schatz, MC, et al. (2012) Nature Biotechnology. doi:10.1038/nbt.2280
3. Bandage [https://rrwick.github.io/Bandage/](https://rrwick.github.io/Bandage/)
4. Busco [http://busco.ezlab.org/](http://busco.ezlab.org/)

## Metagenomics 

Slides - [metagenomics.pptx](metagenomics.pptx)

References and Links:

1. FASTQC [http://www.bioinformatics.babraham.ac.uk/projects/fastqc/](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
2. Trimmomatic [http://www.usadellab.org/cms/?page=trimmomatic](http://www.usadellab.org/cms/?page=trimmomatic)
3. MEGAN 6 [http://ab.inf.uni-tuebingen.de/software/megan6/](http://ab.inf.uni-tuebingen.de/software/megan6/)
4. Kraken [https://ccb.jhu.edu/software/kraken/](https://ccb.jhu.edu/software/kraken/)
5. QIIME [http://qiime.org/](http://qiime.org/)
6. PhyloSIFT [https://phylosift.wordpress.com/](https://phylosift.wordpress.com/)
7. IDBA-UD: Peng, Y., Leung, H. C., Yiu, S. M., & Chin, F. Y. IDBA-UD: a de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth. Bioinformatics, 28(11), 1420-1428, 2012.
8. Meta-IDBA: Peng Y, Leung HCM, Yiu SM, Chin FYL: Meta-IDBA: a de Novo assembler for metagenomic data. Bioinformatics 2011, 27:i94-i101.
9. Velvet: Zerbino DR, Birney E. Velvet: algorithms for de novo short read assembly using de Bruijn graphs. Genome Res. 2008 May;18(5):821-9.
10. MetaVelvet: Namiki, T., Hachiya, T., Tanaka, H., & Sakakibara, Y. MetaVelvet: an extension of Velvet assembler to de novo metagenome assembly from short sequence reads. Nucleic acids research, 40(20), e155-e155, 2012.
11. Ray: Boisvert, S, Raymond, F, Godzaridis, Ã‰, Laviolette, F, Corbeil, J. Ray Meta: scalable de novo metagenome assembly and profiling. Genome biology, 13(12), R122, 2013.
12. LAP: Ghodsi M, Hill CM, Astrovskaya I, Lin H, Sommer DD, Koren S, Pop M. De novo likelihood-based measures for comparing genome assemblies. BMC research notes 6:334, 2013.
13. ALE: Clark, SC, Egan, R, Frazier, PI, Wang, Z. ALE: a generic assembly likelihood evaluation framework for assessing the accuracy of genome and metagenome assemblies. Bioinformatics, 29(4) 435-443, 2013.
14. QUAST: Gurevich, A, Saveliev, V, Vyahhi, N, Tesler, G. QUAST: quality assessment tool for genome assemblies. Bioinformatics, 29(8), 1072-1075, 2013.
15. FRCbam: Vezzi, F, Narzisi, G, Mishra, B. Reevaluating assembly evaluations with feature response curves: GAGE and assemblathons. PloS ONE, 7(12), e52210, 2013.
16. REAPR: Hunt, M, Kikuchi, T, Sanders, M, Newbold, C, Berriman, M, & Otto, TD. REAPR: a universal tool for genome assembly evaluation. Genome biology, 14(5), R47, 2013.
17. KEGG [http://www.genome.jp/kegg/](http://www.genome.jp/kegg/)
18. EBI Metagenomics - [https://www.ebi.ac.uk/metagenomics/](https://www.ebi.ac.uk/metagenomics/)
19. MG-RAST [http://metagenomics.anl.gov/](http://metagenomics.anl.gov/)
20. IMG/M [https://img.jgi.doe.gov/cgi-bin/m/main.cgi](https://img.jgi.doe.gov/cgi-bin/m/main.cgi)
21. ClovR [http://clovr.org/](http://clovr.org/)
22. prokka [http://www.vicbioinformatics.com/software.prokka.shtml](http://www.vicbioinformatics.com/software.prokka.shtml)
23. MetAMOS [http://metamos.readthedocs.io/en/v1.5rc3/](http://metamos.readthedocs.io/en/v1.5rc3/)
24. Trinity [https://github.com/trinityrnaseq/trinityrnaseq/wiki](https://github.com/trinityrnaseq/trinityrnaseq/wiki)
25. Cufflinks [http://cole-trapnell-lab.github.io/cufflinks/](http://cole-trapnell-lab.github.io/cufflinks/)
26. Bioconductor [https://www.bioconductor.org/](https://www.bioconductor.org/)
27. Cytoscape [http://www.cytoscape.org/](http://www.cytoscape.org/)
28. MeV [http://www.tm4.org/mev.html](http://www.tm4.org/mev.html)
